### DISCUSSION TITLE
All items ordered across all orders

### DISCUSSION THREAD

**Discussion by pipeline-eric** [etsy/open-api#737] [Q&A]
I'm trying to create an app that will download the data as JSON so that I can see all items ordered across all orders. Is this the best route to take or am I missing something? Has anyone else done something similar?

**Comment by etsyachristensen** [comment]
@pipeline-eric This is a pretty wide open comment. Are you looking to get all orders in your own shop? You can't get all items across orders not associated with your own shop so I assume that's the case. But there's no endpoint that will return all individual products ordered across multiple receipts. You'd have to get all receipts and go through the transactions in each receipt to get the listing ids. Then you can call the getListingsByListingIds endpoint to get the products.

**Reply by etsyachristensen** [reply]
If you'd like to register a feature request for an endpoint that will like all products in receipts (based on a list of receipt ids, date range, etc) you can go to the Ideas category in the discussion board and post there.

**Reply by pipeline-eric** [reply]
All orders in my own shop so I can get a combined list to see that I need to make x product 1, y product 2 etc. to fulfill the current orders.

**Reply by etsyachristensen** [reply]
Unfortunately there's nothing in the system for this. As I mentioned you can post a request for this as a feature in the Idea category. Otherwise you'll have to loop through all of your receipts and transactions to pull the listing ids. From there, get the products.
